### ISSUE TITLE
DAR401 reraise from library, multi except

### ISSUE THREAD

**Issue by Dreamsorcerer** [terrencepreilly/darglint#102]
Ah, unfortunately, my code still seems to trip this up, by having multiple except blocks:

```
async def error_middleware(overrides: Mapping[int, _Handler], request: web.Request,
                           handler: _Handler) -> web.StreamResponse:
    """Use custom handlers for error conditions.

    Args:
        overrides: A dictionary of status codes mapping to handlers.
        request: The current request object.
        handler: The handler that should be run.

    Returns:
        Response from override, if status code in overrides, otherwise
        return the response from handler.

    Raises:
        web.HTTPException: Reraises any HTTPExceptions without an override.
    """
    try:
        return await handler(request)
    except web.HTTPException as e:
        override = overrides.get(e.status)
        if override is not None:
            return await override(request)

        raise
    except Exception:
        logging.exception("Unknown error")
        return await handle500(request)
```

Gives me:
```
middlewares.py:39:1: DAR401 Missing exception(s) in Raises section: -r Exception
middlewares.py:51:1: DAR402 Excess exception(s) in Raises section: +r web.HTTPException
```

So, it's now reporting that the web.HTTPException is not being raised and that Exception is being raised, which are both incorrect.

_Originally posted by @Dreamsorcerer in https://github.com/terrencepreilly/darglint/issues/88#issuecomment-636531012_
